main.go

```diff
@@ -63,6 +67,26 @@ func main() {
 		return commands.GetEthereumAddress(unitID)
 	})
 
+	// define environment
+	var env gateways.Environment
+	var confFileName string
+	if ENV == "prod" {
```

Comment:
Does this mean we need to update build release GA to set this?

Reply:
What is GA stands for?

Reply:
github actions:
eg here do we need to apss a flag:
`https://github.com/DIMO-Network/edge-network/blob/21ea0e0801642d9f0fa89b92720fe9457eddf260/.github/workflows/release.yaml#L22`